src/WorldView.h

```diff
@@ -132,6 +132,14 @@ class WorldView: public UIView {
 	void OnPlayerDockOrUndock();
 	void OnPlayerChangeTarget();
 	void OnPlayerChangeFlightControlState();
+	/**
+	 * Handler for "requestTimeAccelerationInc" event
+	 */
+	void OnRequestTimeAccelInc();
+	/**
+	 * Handler for "requestTimeAccelerationDec" event
+	 */
+	void OnRequestTimeAccelDec();
```

Comment:
Last nitpick: I'd prefer 1 line C++ style comment rather than 3-line C style. I don't know what the others say (speak up if in disagreement).